app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -105,11 +102,11 @@ const MapImpl: React.FunctionComponent = () => {
   //if there are no states loaded
   if (state.features.length === 0) {
     //if the data variable has been loaded
-    if (initialFeaturesQueryResult.data) {
+    if (keplerState.map) {
       // Not tracking any features yet, add the boroughs we loaded
       dispatch(
         addMapFeatures(
-          initialFeaturesQueryResult.data.features.map(feature => ({
+          stateJSON.map(feature => ({
```

Comment:
In general I'd prefer to encapsulate this, but it's OK for now.
https://www.apollographql.com/docs/react/caching/cache-interaction/